configure

```diff
@@ -3115,6 +3129,22 @@ case $_backend in
 		;;
 esac
 
+#
+# In raspberry Pi, we don't use find_sdlconfig since we could be crosscompiling, but still we use SDL
+#
+case $_backend in
+	raspberrypi)
+		INCLUDES="$INCLUDES -I$RPI_ROOTDIR/usr/include/SDL -D_GNU_SOURCE=1 -D_REENTRANT"
+		LIBS="$LIBS -L$RPI_ROOTDIR/usr/lib -L$RPI_ROOTDIR/usr/lib/arm-linux-gnueabihf -lSDL"
+		DEFINES="$DEFINES -DSDL_BACKEND"
+		add_line_to_config_mk "SDL_BACKEND = 1"
+		_16bit=yes
+		_savegame_timestamp=no
```

Comment:
Don't set `_savegame_timestamp` here -- set it in the case just before this, which already disables timestampes for `gph | n64`.